setup/configuration/configuration.go

```diff
@@ -96,6 +94,11 @@ func NewClient(term terminal.Terminal, kubeconfigPath string) (client.Client, *r
 		term.Fatalf(err, "cannot create client config")
 	}
 
+	// Set QPS and Burst to higher values to avoid client-side throttling issues
+	// prometheus uses these QPS and Burst values so it shouldn't be an issue, see https://github.com/prometheus-operator/prometheus-operator/blob/9d68ecf289d711c66bef39d2f83429265abc6986/pkg/k8sutil/k8sutil.go#L96-L97
+	clientConfig.QPS = 100
+	clientConfig.Burst = 100
```

Comment:
Good catch.
Would be worth changing that in e2e test init part as well - I don't think that we set anything explicitly there.
Also, for clients created for ToolchainClusters CRs we set 20/30 https://github.com/codeready-toolchain/toolchain-common/blob/master/pkg/cluster/service.go#L32-L33 but I'm not sure if it can have a negative impact on anything we use the clients for :thinking:

Reply:
Thanks! Good point about e2e tests. Do you know if we see the client-side throttling messages there? I can look into it if you're not sure.

>but I'm not sure if it can have a negative impact on anything we use the clients for

I'm not sure but I doubt there are concerns on the client-side except I assume that if the server receives too many requests then it could hit server-side throttling. I think we need to be more careful in production code to avoid too many clients with high QPS/Burst so that we reduce the likelihood of bombarding the api server with too many requests too quickly and cause performance issues. But it doesn't seem like we're doing that as a regular practice so I guess we should be fine. It's likely a lesser concern in our tests since the only api server requests are coming solely from the tests and operators whereas on production there are users & workloads as well that are far more unpredictable.